exercises/largest-series-product/cases_test.go

```diff
@@ -20,10 +20,10 @@ var tests = []struct {
 	{"73167176531330624919225119674426574742355349194934", 6, 23520, true, ""},
 	{"0000", 2, 0, true, ""},
 	{"99099", 3, 0, true, ""},
-	{"123", 4, -1, false, "span must be smaller than string length"},
+	{"123", 4, -1, false, "string length must be greater than or equal to span"},
```

Comment:
This is correct, but it sounds sort of back to front. Would it be better to say:

> span must be smaller than or equal to string length

Reply:
"smaller" is a bit of a problem, I think, as it is primarily used as a dimensionality measurement, not a magnitude measurement. I prefer "less than" or "greater than", and the order is interchangeable from my perspective. Thoughts?